cli/common/common.go

```diff
@@ -73,16 +73,33 @@ func GetBoard(firmwareIndex *firmwareindex.Index, fqbn string) *firmwareindex.In
 	return board
 }
 
-// GetUploadToolDir is an helper function that downloads the correct tool to flash a board,
+// DownloadRequiredToolsForBoard is an helper function that downloads the correct tool to flash a board,
 // it returns the path of the downloaded tool
-func GetUploadToolDir(packageIndex *packageindex.Index, board *firmwareindex.IndexBoard) *paths.Path {
-	toolRelease := indexes.GetToolRelease(packageIndex, board.Uploader)
+func DownloadRequiredToolsForBoard(packageIndex *packageindex.Index, board *firmwareindex.IndexBoard) *paths.Path {
```

Comment:
Yes, this function is already tested since it's a wrapper for `downloads.DownloadTool` that has a unit-test AFAICS https://github.com/arduino/arduino-fwuploader/blob/61f34516038b161fef968ea6f320fdf66e93b9ce/indexes/download/download_test.go#L53-L129